### PR TITLE
COMMERCE-9331 Shipments - Custom fields

### PR DESCRIPTION
**Pull request by baharturk**
https://issues.liferay.com/browse/COMMERCE-9331
https://issues.liferay.com/browse/COMMERCE-9608

@alessiorendina

### CODE REVIEW

**Comment by baharturk** [comment]
ci:test:relevant

File: modules/apps/commerce/commerce-product-test/src/testFunctional/macros/CommerceShipments.macro [hunk 1]
```diff
@@ -24,4 +24,18 @@ definition {
 			value1 = "${quantityInShipment}");
 	}
 
+	macro editCustomFieldTextAndInputAreaInShipments {
```
Comment (alessiorendina) [inline modules/apps/commerce/commerce-product-test/src/testFunctional/macros/CommerceShipments.macro:27]: Rename it to `editShipmentCustomFields`

File: modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CPCommerceShipments.testcase [hunk 2]
```diff
@@ -438,6 +438,302 @@ definition {
 			locator1 = "WorkflowMetrics#SEARCH_FIELD_MAPPINGS_METRICS_INDEXES");
 	}
 
+	@description = "This is a test for COMMERCE-9331 Shipments - Custom fields"
+	@priority = "3"
+	test CanAddACustomFieldToShipmentsAndEditTheValue {
```
Comment (alessiorendina) [inline modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CPCommerceShipments.testcase:443]: Rename it to `CanEditShipmentCustomFields`

File: modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CPCommerceShipments.testcase [hunk 3]
```diff
@@ -438,6 +438,302 @@ definition {
...
+
+			CommerceShipments.editCustomFieldTextAndInputAreaInShipments(
+				customFieldType = "Custom Fields",
+				textValue = "Test 4");
+		}
+	}
+
+	@description = "This is a test for COMMERCE-9331 Shipments - Custom fields"
+	@priority = "3"
+	test CanDeleteCustomFieldsFromShipments {
```
Comment (alessiorendina) [inline modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CPCommerceShipments.testcase:584]: Rename it to `CanDeleteShipmentCustomFields`

File: modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CPCommerceShipments.testcase [hunk 4]
```diff
@@ -438,6 +438,302 @@ definition {
...
+				value1 = "1");
+
+			Button.clickSave();
+
+			Click(locator1 = "Icon#CLOSE");
+
+			Refresh();
+		}
+
+		task ("And goes to the Custom Fields tab, writes inside the input area") {
```
Comment (alessiorendina) [inline modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CPCommerceShipments.testcase:691]: It's not an input area, rename it to `text input field`

**Review by alessiorendina — CHANGES_REQUESTED** [review]
@baharturk please check my comments and move commits that contain autogenerated files at the end of the history.

The commits to be moved at the end are:

**COMMERCE-9331 Build lang
COMMERCE-9331 Regen REST module
COMMERCE-9331 Regen services
COMMERCE-9331 Semantic versioning (gw baseline)**

**Comment by alessiorendina** [comment]
Base PR: https://github.com/liferay-commerce/liferay-portal/pull/2838

**Comment by baharturk** [comment]
ci:test:sf

**Comment by baharturk** [comment]
ci:test:relevant

**Pull request closed without merging**